samples/nrf_desktop/src/modules/ble_adv.c

```diff
@@ -128,42 +116,108 @@ static int ble_adv_stop(void)
 	return err;
 }
 
-static int ble_adv_start(void)
+static void bond_scan(const struct bt_bond_info *info, void *user_data)
```

Comment:
For the same reason as above, maybe `bond_find()` or `bonded_peer_find()`?

Reply:
`bond_find` it is